trampoline/pom.xml

```diff
@@ -78,6 +78,11 @@
 			<artifactId>org.eclipse.jgit</artifactId>
 			<version>4.8.0.201706111038-r</version>
 		</dependency>
+		<dependency>
```

Comment:
This will need to be added to the build.gradle file as well. @ErnestOrt Is there are reason we need both files?

Reply:
@JTWalraven The reason is because Trampoline fully supports Gradle and Maven, which means Trampoline needs to be able to be started using gradle wrapper provided.

Reply:
Ah, I see. You mean for when you run the projects from Trampoline. Maybe there should actually be a simple demo project for the build types whereas Trampoline uses just the gradle build script? Either way is fine, it may just be easier to manage.